backends/occa/ceed-occa-restrict.c

```diff
@@ -57,7 +57,7 @@ int CeedElemRestrictionApply_Occa(CeedElemRestriction r,
   if (!strided) {
     ierr = CeedElemRestrictionGetCompStride(r, &compstride); CeedChk(ierr);
   }
-  const CeedTransposeMode restriction = (tmode == CEED_NOTRANSPOSE);
+  bool CeedTransposeMode restriction = (tmode == CEED_NOTRANSPOSE);
```

Comment:
These two OCCA changes were to silence some warnings that ICC threw.

Reply:
This can't possibly be right. Perhaps `bool restriction = ...`?

Reply:
Right. Fixed in one of the various fixup I added.